server/data/restClient.ts

```diff
@@ -1,27 +1,24 @@
 import { Readable } from 'stream'
-import superagent from 'superagent'
+
 import Agent, { HttpsAgent } from 'agentkeepalive'
+import superagent from 'superagent'
 
 import logger from '../../logger'
 import sanitiseError from '../sanitisedError'
-import { ApiConfig } from '../config'
+import type { ApiConfig } from '../config'
 import type { UnsanitisedError } from '../sanitisedError'
 import { restClientMetricsMiddleware } from './restClientMetricsMiddleware'
 
-interface GetRequest {
-  path?: string
-  query?: string
+interface Request {
+  path: string
+  query?: object | string
```

Comment:
think this will need to be `Record<string, unknown>` otherwise you won't be able to pass anything other than empty object?

Reply:
`Record<string, unknown>` "satisfies" `object`. i just used the official type accepted by the superagent function:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8139a88dd28ea580afedc673c18ceb24df95effb/types/superagent/index.d.ts#L166